cardano-crypto-class/src/Cardano/Crypto/KES/Class.hs

```diff
@@ -66,7 +66,10 @@ class ( Typeable v
 
   decodeSigKES :: Decoder s (SigKES v)
 
-  genKeyKES :: MonadRandom m => Natural -> m (SignKeyKES v)
+  genKeyKES :: Seed -> Natural -> SignKeyKES v
+
+  -- | The upper bound on the 'Seed' size needed by 'genKeyKES'
+  seedSizeKES :: proxy v -> Natural -> Natural
```

Comment:
Maybe document what the `Natural` argument is, it's not clear to me.

EDIT: I found out the argument is removed in a later commit.

(Using a newtype or type synonym instead of `Natural` throughout this module would make it much more self-documenting, IMO. Is `Natural` a size, the current period, the max number of periods, ...?)

Reply:
https://github.com/input-output-hk/cardano-base/pull/113, type alias `Period` used